packages/php-storage-driver-common/proto/info.proto

```diff
@@ -85,8 +93,9 @@ message TableInfo {
   repeated string primaryKeysNames = 4; // primary key columns names
   int64 rowsCount = 5;
   int64 sizeBytes = 6;
-  google.protobuf.Any meta = 7; // metadata specific for each backend
+  TableType tableType = 7;
```

Comment:
Pohravam sa s mylienkou, by to nemohol byt normalne dalsi objectType teraz tam je 
```
enum ObjectType {
  DATABASE = 0;
  SCHEMA = 1;
  TABLE = 2;
  VIEW = 3;
}
```
tak by tam pribudol len 
```
enum ObjectType {
  DATABASE = 0;
  SCHEMA = 1;
  TABLE = 2;
  VIEW = 3;
  EXTETRNAL_TABLE = 4;
}
```

Reply:
Ale je to asi spravnejsie takto aj v doc je to tak rozdelene https://cloud.google.com/bigquery/docs/external-tables 
![Uploading Screenshot 2024-04-17 at 10.29.20.png…]()